### DISCUSSION TITLE
bbox 갯수가 적은 데이터만 학습

### DISCUSSION THREAD

**Discussion by kmouleejunhyuk** [boostcampaitech2/object-detection-level2-cv-06#11] [ideas]
bbox갯수가 5개 이하인 데이터만 추출해 학습을 진행했습니다.  
실험 결과
#26

**Comment by ppskj178** [comment]
이런 작업들을 할때 json 파일 자체를 수정해서 진행하면 여러모로 편라할 것 같은데.
ex) [기본] - train.json, [bbox < 5] - bbox_under_5.json, [area > 10000] - area_up_10000.json
[위와 같이 작성해서 dataset에 해당  json을 인자로 넘겨주는 방식]

기존 json을 읽어서 pandas 등으로 여러 가공을 한 후에 새 json 파일로 포팅해주는 프로그램이 있으면 좋을 듯 합니다.

제 EDA랑 Viewer 조금 수정하면 금방 만들 수 있을 것 같은데

저도 이제는 모델을 좀 봐야될것 같아서요...

혹시 연습 겸 해보실 분 계실까요?

없으면 모델 연습 좀 해보고 나중에 제가 해보겠습니다.

**Reply by ppskj178** [reply]
> 제가 어차피 area에 대한 부분을 진행할거라 준혁님께서 EDA 해주신 파일을 참고해서 시도해보겠습니다

https://github.com/boostcampaitech2/object-detection-level2-cv-06/issues/17#issue-1012944184
관련해서 연습겸 이슈로 포팅해봤습니다.

**Reply by kmouleejunhyuk** [reply]
@ppskj178  기존 json을 읽어서 기준에 따라 새 json을 만드는 파일을 LJH_EDA/data2easy.ipynb에 업로드했습니다. 참고하시면 될듯요

**Reply by ppskj178** [reply]
@kmouleejunhyuk
https://github.com/boostcampaitech2/object-detection-level2-cv-06/issues/17
해당 기능 이슈로 작성했는데 댓글에는 자동으로 안나 오네요.
이슈화 하면 댓글로 달아줘야겠어요.
재환님이 작업해주신다고 합니다.
